nevergrad/optimization/optimizerlib.py

```diff
@@ -2018,46 +2040,46 @@ def __init__(self, parametrization: IntOrParameter, budget: tp.Optional[int] = N
                         optimClass = SQP
                     else:
                         # This is the realm of population control. FIXME: should we pair with a bandit ?
-                        optimClass = TBPSA  # type: ignore
+                        optimClass = TBPSA
                 else:
                     if self.has_discrete_not_softmax or not self.parametrization.descriptors.metrizable or not self.fully_continuous:
-                        optimClass = DoubleFastGADiscreteOnePlusOne  # type: ignore
+                        optimClass = DoubleFastGADiscreteOnePlusOne
                     else:
                         if num_workers > budget / 5:
                             if num_workers > budget / 2. or budget < self.dimension:
-                                optimClass = MetaTuneRecentering  # type: ignore
+                                optimClass = MetaTuneRecentering
                             elif self.dimension < 5 and budget < 100:
-                                optimClass = DiagonalCMA  # type: ignore
+                                optimClass = DiagonalCMA
                             elif self.dimension < 5 and budget < 500:
-                                optimClass = Chaining([DiagonalCMA, MetaModel], [100]).set_name("parachaining", register=True)  # type: ignore
+                                optimClass = Chaining([DiagonalCMA, MetaModel], [100])
```

Comment:
@teytaud names and registration are *only* required for using an optimizer through the registry. This is not the case here, so better not register it